roles/edpm_reboot/tasks/main.yaml

```diff
@@ -65,16 +65,16 @@
 
 - name: Print message if reboot is required, but is not going to be started
   ansible.builtin.debug:
-    msg: "Reboot is required but was not started. User has to plan the reboot and set edpm_reboot_force_reboot to true"
-  when:
-    - reboot_required|default(false)
-    - nova_conf_check.stat.exists
-    - not edpm_reboot_force_reboot|bool
+    msg: "Reboot is required but was not started. Edpm_reboot_strategy is set to never or this is
+          already deployed machine and reboot has to be planned. To start reboot set edpm_reboot_strategy 
```

Comment:
the already deployed check only considers the existence of nova.conf, so perhaps that should be the description. either that, or we need to add a way to flag an edpm node as already deployed independent of nova as there could be other node types.

Reply:
I discussed it with Jiri and he mentioned networker nodes with ovn. I'm going to add check for ovn files on nodes to include networker nodes. Do you know other node types and check for those?

Reply:
@slagle In last revision I made a change that makes the check independent of nova conf. Please check pull request once again